curator/install.sh

```diff
@@ -2,7 +2,8 @@
 
 set -ex
 
-rpm -q epel-release || yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+yum makecache fast
```

Comment:
I don't know that you should add this - we generally try not to in our Dockerfiles